modules/internal/ChapelBase.chpl

```diff
@@ -1131,6 +1125,7 @@ module ChapelBase {
     return ir;
   }
   
+  pragma "compiler generated" 
   pragma "donor fn"
   pragma "auto copy fn"
   inline proc chpl__autoCopy(x) return chpl__initCopy(x);
```

Comment:
What requirements? I thought compiler generated just changed the resolution priority, so that other candidates are preferred to ones marked "compiler generated". If there are other requirements, could you explain?